doc/src/manual/profile.md

```diff
@@ -312,3 +312,35 @@ memory allocation). The recommended procedure is to force compilation by executi
 you want to analyze, then call [`Profile.clear_malloc_data()`](@ref) to reset all allocation counters.
  Finally, execute the desired commands and quit Julia to trigger the generation of the `.mem`
 files.
+
+# External Profiling
+
+Currently Julia supports `Intel VTune`, `OProfile` and `perf` as external profiling tools.
+The interfaces are all maintained as LLVM modules, and they support code annotations.
+
+Depending on the tool you choose, compile with `USE_INTEL_JITEVENTS`, `USE_OPROFILE_JITEVENTS` and
+`USE_PERF_JITEVENTS` set to 1 in `Make.user`. Multiple flags are supported.
+
+Also before running Julia set the environment variable `ENABLE_JITPROFILING` to 1.
+
+Now you have a multitude of ways to employ those tools!
+For example with `OProfile` you can try a simple recording :
+
+```
+>sudo operf -Vdebug ./julia test/fastmath.jl
```

Comment:
I think having the environment variables as part of the script so that `copy-paste` just works, would be good